articles/storage/blobs/storage-lifecycle-management-concepts.md

```diff
@@ -428,6 +428,9 @@ For data that is modified and accessed regularly throughout its lifetime, snapsh
 **I created a new policy, why do the actions not run immediately?**  
 The platform runs the lifecycle policy once a day. Once you configure a policy, it can take up to 24 hours for some actions to run for the first time.  
 
+**If I update an existing policy, How long does it take for the actions to run ?**  
+Note that the updated policy will take up to 24 hours to take effect. Once the policy take effect, it could take up to 24 hours for the actions to run. So overall the policy may take up to 48 hours to execute.   
```

Comment:
Edits:

**If I update an existing policy, how long does it take for the actions to run ?**  
The updated policy takes up to 24 hours to go into effect. Once the policy is in effect, it could take up to 24 hours for the actions to run. Therefore, the policy may take up to 48 hours to execute.